Resources/input/fsh/profiles/GEM_ERPCHRG_PR_Consent.fsh

```diff
@@ -5,20 +5,20 @@ Id: GEM-ERPCHRG-PR-Consent
 * insert MetaProfile(GEM_ERPCHRG_PR_Consent)
 
 * status = #active (exactly)
-* scope.coding 1..1
+* scope.coding 1..1 MS
   * system = $cs-consent-scope (exactly)
   * code = #patient-privacy (exactly)
   * display = "Privacy Consent" (exactly)
 
-* category 1..1
+* category 1..1 MS
 * category from GEM_ERPCHRG_VS_ConsentType (required)
-* patient 1..1
-  * identifier 1..1
+* patient 1..1 MS
+  * identifier 1..1 MS
   * identifier only IdentifierKvid10
     * assigner MS
     * assigner.identifier MS
     * assigner.identifier only IdentifierIknr
-* dateTime 1..1
+* dateTime MS
```

Comment:
Warum wird hier die Kardinalität nicht mit angegeben?
> * dateTime 0..1 MS 
Würde ich als deutlich angenehmer zu lesen finden, oder hat das direkte auswirkungen auf die Darstellung in simplifier?

Reply:
Das kann man machen. Es ist nur eine "überflüssige" Angabe, da man nichts ändert. 
Kann ich aber noch einfügen, wie du meinst